src/app/components/input/input.component.html

```diff
@@ -1 +1,18 @@
-<input type="text" class="stl-input" [ngModel]="model" (ngModelChange)="modelValueChange($event)" />
+<div class="stl-input"
```

Comment:
I have a suggestion:
```suggestion
<div class="stl-input-component"
```
Let's use class names for the root element of a component always ends with "-component", while all the rests CSS classes **must not** ends with "-components". Typical SCSS file then looks like:
```
._mycomponentname_-component {
    .any-class-name-here {
    }
}
```
In that case, we can use any class name for _any-class-name-here_, without worrying possible intersection with the same class name in another component. This convention is simple (actually, might be simpler, than use CSS Modules, which are limited functionality and sometimes buggy IMHO).